### PR TITLE
Implement join the community

### LINKED ISSUE #13: Join the community
This feature is completed when:

- [ ] It resemble the design of the Mobile and Desktop attached screens  for illustration only, see invision/zeplin (it does not have to be pixel perfect at this stage)
- [ ] It has all the text exported into the language files (even if dummy text)
- [ ] Create an empty "click" handler ( newsletter subscription in another ticket)
- [ ] It enhance the current look and feel of 'resources\js\components\sections\welcomeJoin.vue'

Invision designs: https://projects.invisionapp.com/d/main/default/#/console/17236584/446551990/preview
zeplin: https://app.zeplin.io/project/6047d3685cdff106a3f16337/screen/6047d80ebafdbb06f6d2df8e

Note the CSS is being written using tailwind. it is suggested to develop the HTML in the live editor first (https://play.tailwindcss.com/) and them move it over.

### PR DESCRIPTION
**Pull request by sosualfred**
It is not perfect but It is quite close to the original design. Below are screenshots of the implementation. 

fixes #13

DESKTOP:
image

MOBILE: 
image

### CODE REVIEW

File: resources/js/components/sections/welcomeJoin.vue [hunk 1]
```diff
@@ -1,17 +1,39 @@
 <template>
-  <div class="welcomeSearch relative flex flex-wrap items-center justify-between">
-    <div class="container px-4 mx-auto my-12 flex flex-wrap items-center justify-between">
-      welcomeJoin
+  <main class="bg-gray-600 py-14 mb-10">
```
Comment (Zelig880) [inline resources/js/components/sections/welcomeJoin.vue:2]: Use of "main" for this section is incorrect. 

Use "article" instead

Comment (Zelig880) [inline resources/js/components/sections/welcomeJoin.vue:2]: Please use background color: #cf6f5f as shown by the designs.

**Review by Zelig880 — CHANGES_REQUESTED** [review]
Thank you for the PR!

I have left a couple of comments, plus sent you the illustration needed on this banner.

I think you have worked using the designs in the ticket, but if you log into invision,you can see the new ones (luckily for you they are the same, just different colour).

File: resources/js/components/sections/welcomeJoin.vue (hunk 1 above)
Comment (sosualfred) [reply resources/js/components/sections/welcomeJoin.vue]: I have changed the main to article. Whiles, I was implementing this change, I was wondering if using "section" will be better than "article" in this context.

**Comment by sosualfred** [comment]
This is how the new changes look like. :smile: 

image
image

**Review by Zelig880 — APPROVED** [review]

**Pull request merged**